pkg/ebpf/c/tracee.bpf.c

```diff
@@ -915,12 +915,12 @@ BPF_ARRAY(file_filter, path_filter_t, 3);                          // filter vfs
 BPF_PERCPU_ARRAY(bufs, buf_t, MAX_BUFFERS);                        // percpu global buffer variables
 BPF_PROG_ARRAY(prog_array, MAX_TAIL_CALL);                         // store programs for tail calls
 BPF_PROG_ARRAY(prog_array_tp, MAX_TAIL_CALL);                      // store programs for tail calls
-BPF_PROG_ARRAY(sys_enter_tails, MAX_EVENT_ID);                     // store syscall specific programs for tail calls from sys_enter
-BPF_PROG_ARRAY(sys_exit_tails, MAX_EVENT_ID);                      // store syscall specific programs for tail calls from sys_exit
-BPF_PROG_ARRAY(sys_enter_submit_tail, MAX_EVENT_ID);               // store program for submitting syscalls from sys_enter
-BPF_PROG_ARRAY(sys_exit_submit_tail, MAX_EVENT_ID);                // store program for submitting syscalls from sys_exit
-BPF_PROG_ARRAY(sys_enter_init_tail, MAX_EVENT_ID);                 // store program for performing syscall tracking logic in sys_enter
-BPF_PROG_ARRAY(sys_exit_init_tail, MAX_EVENT_ID);                  // store program for performing syscall tracking logic in sys_exits
+BPF_PROG_ARRAY(sys_enter_tails, 10240);                     // store syscall specific programs for tail calls from sys_enter
```

Comment:
These are missing alignment in comments. Also, why do you think MAX_EVENT_ID is wrong here ? This is a tail call map and we could, at most, have 1 tail call, in each map, for each existing event, no ?

Reply:
sorry, this is unrelated to this PR, i'll revert back.
its just a quick fix of this bug #2202 so i could test something.